admin/test-artifacts

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+# Copyright 2015 ClusterHQ Inc.  See LICENSE file for details.
+"""
+Check that published artifacts can be installed.
+"""
+import argparse
+import os
+import unittest
+
+from flocker.common.version import make_rpm_version
+from flocker.provision import PackageSource
+
+from admin.client import (
+    DockerContainer, get_steps_pkg, get_steps_pip, run_steps
+)
+
+
+class TestClientPackages(unittest.TestCase):
+
+    def __init__(self, method, package_source=PackageSource()):
+        unittest.TestCase.__init__(self, method)
+        self.package_source = package_source
+
+    def install(self, distribution, get_steps):
+        steps = get_steps(distribution, self.package_source)
+        container = DockerContainer.from_distribution(distribution)
+        filename = '{}.out'.format(self.id())
+        with open(filename, 'w') as out:
+            status = run_steps(container, steps, out)
+            self.assertEqual(
+                status, 0,
+                'Exit status = {}. See file {!r}'.format(status, filename))
+        os.remove(filename)
+
+    def test_centos_7_rpm(self):
+        """
+        CentOS 7 client packaging works.
+
+        Although CentOS 7 is not a supported client, the client packages
+        get built, so it is installed here to test RPM packaging.
```

Comment:
I have added FLOC-3243 to add more direct tests. Until that issue is resolved, this test adds a lot of value, so I intend to keep it in.

(Also, since the client packages include tools that are useful on nodes (e.g. `flocker-ca`) it would actually be sensible to add client support for CentOS.)